src/Groups/libraries/smallgroups.jl

```diff
@@ -176,6 +176,7 @@ The following functions are currently supported as values for `func`:
 - `is_solvable`
 - `is_supersolvable`
 - `number_conjugacy_classes`
+- `exponent`
```

Comment:
Up to now, the list was ordered alphabetically. What is the reason to put `exponent` here?

Reply:
Just didn't think about it. Moved it up now, thanks!